evalml/objectives/binary_classification_objective.py

```diff
@@ -7,11 +7,7 @@
 
 
 class BinaryClassificationObjective(ObjectiveBase):
-    """Base class for all binary classification objectives.
-
-    problem_types (list(ProblemType)): List of problem types that this objective is defined for. Set to [ProblemTypes.BINARY].
```

Comment:
Why did this get removed?

Reply:
@bchen1116 `problem_types` would have been included twice, once as part of the docstring and once as part of the class attributes
Before: https://evalml.alteryx.com/en/stable/generated/evalml.objectives.BinaryClassificationObjective.html?highlight=BinaryClassificationObjective
After: https://evalml.alteryx.com/en/1653-update-docs-to-show-attributes/generated/evalml.objectives.BinaryClassificationObjective.html?highlight=BinaryClassificationObjective

Reply:
Maybe we can keep `Base class for all binary classification objectives.`? Agree that the arguments here look somewhat out of place.

Reply:
Gotcha, thanks